graphql/server.go

```diff
@@ -269,7 +272,7 @@ func (c *conn) handleSubscribe(in *inEnvelope) error {
 			return nil, err
 		}
 
-		d := diff.Diff(previous, current)
+		d := diff.Diff(computationInput.Previous, current)
```

Comment:
this is getting pretty spooky but I am hopeful we'll fix it all up once we're done this with transformation